webapp/src/ts/services/telemetry.service.ts

```diff
@@ -88,6 +88,7 @@ export class TelemetryService {
           year: date.year,
           month: date.month,
           day: date.date,
+          aggregate_date: new Date().toISOString(),
```

Comment:
Same format as [we do in feedback service](https://github.com/medic/cht-core/blob/dac993b4529cd70fea76cf9678f0c2e02705e7e5/webapp/src/ts/services/feedback.service.ts#L197)